gdpr/gdpr.sp

```diff
@@ -0,0 +1,18 @@
+locals {
+  gdpr_common_tags = {
+    gdpr   = "true"
+    plugin = "aws"
+  }
+}
+
+benchmark "gdpr" {
+  title         = "AWS General Data Protection Regulation (GDPR)"
```

Comment:
I don't think we need the word "AWS" in here since we're already in the AWS Compliance mod.
```suggestion
  title         = "General Data Protection Regulation (GDPR)"
```

In the UI, ideally it'd be great if GDPR is visible on the sidebar too, right now it's cut off, but can you please check if it's visible entirely on the sidebar once we remove "AWS"?